src/orbit.rs

```diff
@@ -109,25 +106,31 @@ impl AuthMethods {
         match self {
```

Comment:
I think you may be able to do `match (self, auth_token)`, although I vaguely remember trying it and failing

Reply:
nice suggestion